README.md

```diff
@@ -78,5 +78,31 @@ Please note, this outline is an approximation for the course schedule. At e-cono
 
 __If__ time permits we will also look into building cross-platform client applications for __iOS__ and __Android__ with __React Native__. 
 
+## Project description (ITU version)
+
+<i>For ITU students. The following is a draft for what a project description could look like in the ITU project base, as an independent ECTS approved project.</i>
+
+<b>Project titel (Dansk):</b> //TODO
+
+<b>Project title (English):</b> Developing Client Side applications with JavaScript and ReactJs
+
+<b>Problem statement:</b> Learn how to develop client side web applications written in JavaScript and using ReactJs library. This is not possible through any other course currently offered at ITU.
```

Comment:
I think this kind of course is already course is already offered, but based on different technologies. How about changing the second part to `These technologies are now widely used in the IT industry, but currently not covered in other courses offered at ITU`